internal/cfgplugins/interface.go

```diff
@@ -884,7 +884,7 @@ func NewAggregateInterface(t *testing.T, dut *ondatra.DUTDevice, b *gnmi.SetBatc
 	aggID := l.LagName
 	agg := l.NewOCInterface(aggID, dut)
 	agg.Type = oc.IETFInterfaces_InterfaceType_ieee8023adLag
-	if deviations.IPv4MissingEnabled(dut) {
+	if !deviations.IPv4MissingEnabled(dut) {
```

Comment:
Good fix here.  I noticed nearly everywhere this deviation is used, it is combined with a second deviation:

```
	if deviations.InterfaceEnabled(dut) && !deviations.IPv4MissingEnabled(dut) {
		s4.Enabled = ygot.Bool(true)
	}
```
Should we follow this pattern here too?